droneml_dialog.py

```diff
@@ -1,16 +1,18 @@
-from qgis.PyQt import QtWidgets
+from pathlib import Path
+from qgis.PyQt import QtWidgets, QtCore
 from qgis.core import QgsRasterLayer, QgsVectorLayer, QgsProject
-import os
-import shapely
-import numpy as np
-import geopandas as gpd
-import rioxarray
-from pyproj import CRS
-from .segmentmytif.src.segmentmytiff.main import make_predictions
-from dask.distributed import LocalCluster, Client
+from segmentmytif.main import read_input_and_labels_and_save_predictions
+from segmentmytif.features import FeatureType
+import logging
 
-FONTSIZE = 16
+# Turn off the logger
+logging.getLogger().setLevel(logging.CRITICAL)
```

Comment:
Why turn it off? Let's discuss this

Reply:
As we discussed, QGIS now raise error when log in enabled. But it's a good idea to show this. I tracked this in #11 .